apps/api/src/user/dto/create.user/create.user.ts

```diff
@@ -0,0 +1,16 @@
+import { IsBoolean, IsString } from "class-validator";
+
+export class ICreateUserDTO {
+    @IsString()
+    name: string;
+    @IsString()
+    email: string;
+    @IsString()
```

Comment:
Make the following changes:
- set `profilePictureUrl`, `name` as optional
- `isActive`, `isOnboardingFinished` to optional, with the default value set to true in the service layer
- `isAdmin` to optional, with the default value set to false in the service layer